docker/entrypoint.sh

```
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Assumes the image was built with:
+# docker build . -f docker/Dockerfile.prod --build-arg NEXT_PUBLIC_BASE_URL=http://bDn05eu06Wxq:80 -t ghcr.io/colinmollenhour/inbox-zero:latest
+
+set -e
+
+# Replace NEXT_PUBLIC_* placeholders with actual runtime values
+if [ -d /app/apps/web/.next ]; then
+  printenv | grep NEXT_PUBLIC_BASE_URL | while read -r line; do
+    key=$(echo "$line" | cut -d "=" -f1)
```

Shellcheck (0.11.0)

[warning] key appears unused. Verify use (or export if used externally).

(SC2034)